app/dashboard/views.py

```diff
@@ -1557,3 +1557,25 @@ def change_bounty(request, bounty_id):
         'result': result
     }
     return TemplateResponse(request, 'bounty/change.html', params)
+
+
+def get_users(request):
+    if request.is_ajax():
+        q = request.GET.get('term')
+        profiles = Profile.objects.filter(handle__icontains=q)
+        results = []
+        for user in profiles:
+            profile_json = {}
+            profile_json['id'] = user.id
+            profile_json['text'] = user.handle
+            profile_json['email'] = user.email
+            profile_json['avatar_id'] = user.avatar_id
+            if user.avatar_id:
+                profile_json['avatar_url'] = user.avatar_url
+            profile_json['preferred_payout_address'] = user.preferred_payout_address
+            results.append(profile_json)
+        data = json.dumps(results)
+    else:
+        data = 'fail'
```

Comment:
What happens when this block of code gets executed? Won't the `formatUser` fail? Secondly, if you are sending it with `application/json`, I think it is better to send something like `{message: 'fail'}`